README.md

```diff
@@ -63,6 +64,11 @@
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [https://mohashyne.github.io/MSalyhu_portfolio/]
```

Comment:
- [ ] Your deployment link provided is not linking to your portfolio project but instead, it links to your ReadME file like so 👇🏾 . Kindly fix this

  ![image](https://github.com/mohashyne/MSalyhu_portfolio/assets/84156714/3bb60227-6f24-4a1e-9187-0cde94ff29af)